int32 room_id
string room_name
float32 battery_level